Source/engine/render/clx_render.cpp

```diff
@@ -77,16 +77,18 @@ struct SkipSize {
 
 DVL_ALWAYS_INLINE DVL_ATTRIBUTE_HOT SkipSize GetSkipSize(int_fast16_t remainingWidth, int_fast16_t srcWidth)
 {
-	if (remainingWidth < 0) {
-		// If `remainingWidth` is negative, `-remainingWidth` is the overrun.
-		const int_fast16_t overrunLines = -remainingWidth / srcWidth;
-		return {
-			static_cast<int_fast16_t>(1 + overrunLines),
-			static_cast<int_fast16_t>(-remainingWidth - srcWidth * overrunLines)
-		};
-	}
-	// If `remainingWidth` is non-negative, then it is 0, meaning we drew a whole line.
-	return { 1, 0 };
+	// If `remainingWidth` is negative, `-remainingWidth` is the overrun.
+	// Otherwise, `remainingWidth` is always 0.
+
+	// Remaining width of 0 (= no overrun) is a common case.
+	// The calculation below would result in the same result.
+	// However, checking for 0 and skipping it entirely turns out to be faster.
+	if (remainingWidth == 0) return { 1, 0 };

```

Comment:
```suggestion
	if (remainingWidth >= 0) return { 1, 0 };
```

Reply:
That uses a different `GetSkipSize` though.

Reply:
> That uses a different `GetSkipSize` though.

oh well didn't check that far 😃